### PR TITLE
Allow json and dictionary as input (#6)

### PR DESCRIPTION
**Pull request by thiago-aixplain**

### CODE REVIEW

File: aixtend/modules/pipeline.py [hunk 1]
```diff
@@ -160,7 +161,14 @@ def run_async(self, data: str, name: str = "pipeline_process"):
...
         }
-        payload = json.dumps({ "data": data })
+        if type(data) == dict:
+            payload = json.dumps(data)
+        else:
+            try:
+                data_json = json.loads(data)
+                payload = data
+            except:
+                payload = json.dumps({ "data": data })
```
Comment (krishnadurai) [inline aixtend/modules/pipeline.py:171]: Should we accept a string formatted json in the first place?

**Review by krishnadurai — COMMENTED** [review]
@thiago-aixplain just a few minor improvements are required.

File: aixtend/modules/model.py [hunk 2]
```diff
@@ -166,7 +167,14 @@ def run_async(self, data: str, name: str = "model_process"):
             'x-api-key': self.api_key,
             'Content-Type': 'application/json'
         }
-        payload = json.dumps({ "data": data })
+        if type(data) == dict:
```
[the remark this answers is not in the document]
Comment (thiago-aixplain) [reply aixtend/modules/model.py:170]: Done

File: aixtend/modules/pipeline.py (hunk 1 above)
Comment (thiago-aixplain) [reply aixtend/modules/pipeline.py:171]: I am just addressing the case since it can happen.

**Review by krishnadurai — APPROVED** [review]

**Pull request merged**